primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/PrimeSelenium.java

```diff
@@ -23,6 +23,9 @@
  */
 package org.primefaces.selenium;
 
+import static org.primefaces.selenium.internal.ConfigProvider.DEPLOYMENT_ADAPTER;
```

Comment:
in general we avoid static imports in PF

Reply:
Done